big_vision/configs/vit_i21k.py

```diff
@@ -112,8 +115,26 @@ def get_config(arg=None):
   config.evals.test = {**eval_common, 'split': 'full[:25_600]'}
   config.evals.val = {**eval_common, 'split': 'full[25_600:51_200]'}
   config.evals.train = {**eval_common, 'split': 'full[51_200:76_800]'}
+
+  # Evaluations on ImageNet1k variants
+  eval_common.update(dict(
+      type='classification_with_labelmap', pp_fn=pp_eval_i1k, data_dir=None))
```

Comment:
You included the configs that use the new `classification_with_labelmap` evaluator, but you did not include the evaluator itself. Please either also include it (and mention that in the PR description), or exclude these changes for now. (same in the bit_i21k.py)

Reply:
excluded from opensourcing.